cmd/manager/main.go

```diff
@@ -99,12 +99,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	cl := mgr.GetClient()
+	catalogClient := catalogclient.NewClient(cl)
```

Comment:
I know NewClient isn't net new in this PR, but given the package is named client, let's consider renaming NewClient to New.

Reply:
Good catch. My editor used to suggest getting rid of names like this, but for some reaons no longer does it. Wondering if I'm missing some tools.

Anyway - I created #422 for this.